test/integration/features/coolstore.feature

```diff
@@ -0,0 +1,161 @@
+@coolstore
+Feature: Cool Store
+  In order to test Minishift under load
+  I need to setup a test environment
+  Using CentOS and 4G of memory
+  After which I will import image streams
+  Finally I will deploy the Cool Store
+  During this deployment I intend to measure memory and CPU usage
+
+  Scenario: Starting Minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --memory 4096 --iso-url https://github.com/minishift/minishift-centos-iso/releases/download/v1.0.0/minishift-centos7.iso" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: Import JBoss imagestreams
+    Given Minishift has state "Running"
+     When executing "oc --as system:admin create -n openshift -f https://raw.githubusercontent.com/jboss-openshift/application-templates/master/jboss-image-streams.json" succeeds
+     Then stdout should contain
+      """
+      imagestream "redhat-openjdk18-openshift" created
+      """
+      
+  Scenario: Import JBoss imagestreams
```

Comment:
Can we rename this scenario or be more precise about JBoss here, the previous scenario seems to be same too. 

This scenario seems to import two imagestreams: 
```
imagestream "fis-java-openshift" created
imagestream "fis-karaf-openshift" created
```